http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpConfigurationOption.java

```diff
@@ -134,7 +137,7 @@
     private static final Duration DEFAULT_CONNECTION_ACQUIRE_TIMEOUT = Duration.ofSeconds(10);
     private static final Duration DEFAULT_CONNECTION_MAX_IDLE_TIMEOUT = Duration.ofSeconds(60);
     private static final Duration DEFAULT_CONNECTION_TIME_TO_LIVE = Duration.ZERO;
-    private static final Duration DEFAULT_TLS_HANDSHAKE_TIMEOUT = Duration.ofSeconds(10);
+    private static final Duration DEFAULT_TLS_HANDSHAKE_TIMEOUT = DEFAULT_CONNECTION_TIMEOUT;
```

Comment:
Good to have (Optional):- Can we mention about this change some where in ChangeLogs or Description saying Default TLS_Handshake changed from 10s to 2s.

Reply:
Yup, will add